### PR TITLE
Add description to change login ID page

### PR DESCRIPTION
**Pull request by nicolaslee2775**
fix authgear/authgear-deploy#48

image
image
image

### CODE REVIEW

**Review by axolotl0212 — APPROVED** [review]

File: pkg/lib/authn/identity/service/service.go [hunk 1]
```diff
@@ -395,6 +395,26 @@ func (s *Service) CheckDuplicated(is *identity.Info) (dupeIdentity *identity.Inf
...
+
+	for _, iden := range loginIDs {
+		if iden.LoginIDKey == loginIDKey {
+			displayID = loginIDToIdentityInfo(iden).DisplayID()
+		}
+	}
+
+	return
+}
+
```
Comment (louischan-oursky) [inline pkg/lib/authn/identity/service/service.go:417]: Actually no need to add this function. We should reuse the existing `func (s *Service) Get(userID string, typ authn.IdentityType, id string) (*identity.Info, error)`.

userID: You got it already.
typ: authn.IdentityTypeLoginID
id: the value of r.Form.Get("x_identity_id")

Of course we need to handle the identity not found error.

**Comment by nicolaslee2775** [comment]
@louischan-oursky Updated.

**Comment by louischan-oursky** [comment]
It crashes when I try to add a phone number (I did not have one previously)

**Comment by nicolaslee2775** [comment]
@louischan-oursky Fixed.

**Pull request merged**